action.yml

```diff
@@ -0,0 +1,36 @@
+name: "cargo.toml fmt"
+author: "Gwen Lg"
+description: "Run TOML formatting change or check on Cargo.toml in GitHub Actions"
+
+inputs:
+  version:
+    description: "Version of Taplo"
+    required: false
+    default: "0.9.3"
+
+  files:
+    description: "Files or patterns to check"
+    required: false
+    default: "Cargo.toml"
```

Comment:
I think it would be more universally useful to make the files required and don't specify a default.